lessons/python/cpu-multiprocessing/dask_tutorial.py

```diff
@@ -0,0 +1,32 @@
+'''
+From: http://distributed.readthedocs.io/en/latest/joblib.html
+'''
+
+
+import distributed.joblib
+# Scikit-learn bundles joblib, so you need to import from
+# `sklearn.externals.joblib` instead of `joblib` directly
+from sklearn.externals.joblib import parallel_backend
```

Comment:
hmm, okay, i'll make a disclaimer, just so I understand, it's the parallel_backend that would need to be imported from joblib not sklearn in Fedora/Debian?